netci.groovy

```diff
@@ -21,7 +21,7 @@ def outerloopPlatforms = ['Windows_NT', 'Ubuntu14.04', 'Ubuntu16.04', 'OSX10.12'
                     }
                 }
 
-                Utilities.setMachineAffinity(newJob, os, 'latest-dev15-5')
+                Utilities.setMachineAffinity(newJob, os, 'latest-dev15-7')
```

Comment:
@mmitche This doesn't seem to get a 15.7 machine. What string do I need to get one so I can build .NET Core 2.1 apps?